nixos/modules/services/mail/stalwart-mail.nix

```diff
@@ -78,6 +78,7 @@ in {
       ];
       config.resource = {
         spam-filter = lib.mkDefault "file://${cfg.package}/etc/stalwart/spamfilter.toml";
+        webadmin = lib.mkDefault "file://${cfg.package}/share/web/webadmin.zip";
```

Comment:
Why do we copy the file into the package if we never do anthing with it? It could also be just it's own package to reduce rebuilds.